charts/sonarqube-dce/templates/sonarqube-application.yaml

```diff
@@ -490,7 +490,13 @@ spec:
           secretName: {{ .Values.searchNodes.searchAuthentication.keyStoreSecret }}
       {{- end }}
       - name: sonarqube
+        {{- if .Values.hostPath.enabled }}
+        hostPath:
```

Comment:
We don't persist the application nodes's storage in persistent volumes. Why do we want to add the hostPath here?